components/SearchBar/styles.ts

```diff
@@ -0,0 +1,36 @@
+import { styled } from '../../styles/stitches.config';
+
+export const Div = styled('div', {
+    maxWidth: '334px',
```

Comment:
Isso aqui não joga contra você no responsive?

Reply:
Creio que pelo contrário.. o tamanho é ajustável até um certo ponto.. não continua aumentando, se for uma trela grandona, vai ter no máximo esse tamanho. Faz sentido?

Reply:
Faz sim, pensei que ele teria um tamanho máximo maior, mas faz total sentido :)